flow/project.py

```diff
@@ -491,13 +519,14 @@ def __init__(self, pre=None, post=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
-        """Eligible, when all pre-conditions are true and at least one post-condition is false,
+    def eligible(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
```

Comment:
This does change the API of a user facing class and method. In this case, I am not sure what to do, but it is worth noting.